webapp_utils/errata.py

```diff
@@ -156,9 +158,13 @@ def _errata_exists(self, errata):
         """Checks if given errata is in db."""
         db_connection = self.db_pool.get_connection()
         with db_connection.get_cursor() as cursor:
-            cursor.execute("""select name from errata
-                              where name = '%s'""" % errata)
-            errata_detail = cursor.fetchone()
+            try:
+                cursor.execute("""select name from errata
+                                  where name = '%s'""" % errata)
```

Comment:
http://initd.org/psycopg/docs/usage.html#the-problem-with-the-query-parameters

"Warning Never, never, NEVER use Python string concatenation (+) or string parameters interpolation (%) to pass variables to a SQL query string. Not even at gunpoint." :-)

Reply:
:cold_sweat: